test/sdf/camera_sensor_l8_l16_builtin.sdf

```diff
@@ -24,6 +24,23 @@
           </noise>
         </camera>
       </sensor>
+      <sensor name="camera2" type="camera">
```

Comment:
``camera_16_bit`` sounds more descriptive. Similarly in the code for sensor pointers, handles and helpers, it would be nice to have descriptive names, like ``cameraId8Bit`` or ``sensorPtrCamera8Bit``